frontend/app/assets/stylesheets/archivesspace/token-input.less

```diff
@@ -304,6 +304,52 @@ div.token-input-dropdown ul li.token-input-selected-dropdown-item {
     content:"\e041";
   }
 
+  // ANW-521
+  // Definitions for icons for subjects, based on the term type 
+  .subject .subject_type_cultural_context:before {
```

Comment:
@avatar382 Forgive me if I'm dense, but looks like you've got duplicated `subject_type_cultural_context:before` here (with different content).

Reply:
Yep, that's a typo! I'll push a fix shortly.